client/utils.c

```diff
@@ -110,6 +123,19 @@ TDNFGetSystemError(
     return dwSystemError;
 }
 
+uint32_t
+TDNFGetCurlError(
```

Comment:
Can we make this function static?

Reply:
We could, but why? No other function in that file is static, so it would be very odd.